Blog/views.py

```diff
@@ -18,8 +19,23 @@ def index(request):
 
 
 class PostListView(generic.ListView):
-    model = Post
     paginate_by = 10
+    template_name = 'Blog/post_list.html'
+    model = Post
```

Comment:
Please move this line back to its original position as there is no benefit of changing this.